snuba/snuba_migrations/spans/0007_spans_add_profile_id.py

```diff
@@ -0,0 +1,61 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set_name = StorageSetKey.SPANS
+local_table_name = "spans_local"
+dist_table_name = "spans_dist"
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Add profile_id column.
+    """
+
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=storage_set_name,
+                table_name=local_table_name,
+                column=Column(
+                    "profile_id",
+                    UInt(
+                        64, Modifiers(nullable=True, codecs=["DoubleDelta", "ZSTD(1)"])
+                    ),
+                ),
+                target=OperationTarget.LOCAL,
+                after="span_id",
+            ),
+            operations.AddColumn(
+                storage_set=storage_set_name,
+                table_name=dist_table_name,
+                column=Column(
+                    "profile_id",
+                    UInt(64),
```

Comment:
On the transactions table, profile_id field is a UUID [field](https://github.com/getsentry/snuba/blob/master/snuba/snuba_migrations/transactions/0018_transactions_add_profile_id.py). This should probably be the same. Can you confirm @phacops ?

Reply:
As per conversation with Pierre offline, profile_id needs to be UUID field. Let's change it accordingly.

Reply:
Indeed, it should be a `Uuid` column.

Reply:
ok fixed it.